lld/COFF/Writer.cpp

```diff
@@ -696,6 +699,7 @@ void Writer::run() {
       writeHeader<pe32_header>();
     }
     writeSections();
+    changeLoadConfig();
```

Comment:
Should this run even if `dependentloadflag` is not set?

Reply:
Not really, it is addressed in the latest revision.